pyproject.toml

```diff
@@ -101,5 +101,5 @@ module = "docs.*"
 ignore_errors = true
 
 [build-system]
-requires = ['setuptools>=65.4.1', 'wheel', 'Cython', "poetry-core>=1.0.0"]
+requires = ['setuptools>=65.4.1', 'wheel', 'Cython>=0.29.32,<3.1.0', "poetry-core>=1.0.0"]
```

Comment:
The [error](https://github.com/Bluetooth-Devices/dbus-fast/pull/264#issuecomment-2334550288) was hit with Cython version `0.29.37`, so I don't think this is accurate.

Reply:
Yeah it looks like we need 3+

Reply:
https://github.com/Bluetooth-Devices/dbus-fast/pull/311